cypress/interceptors/response-validation.spec.js

```diff
@@ -0,0 +1,66 @@
+const { assertSchema } = require('@cypress/schema-tools');
```

Comment:
In general, it would be beneficial to add some UI-side verification to the test to represent the e2e flow we can do with interceptors. What do you think?

Reply:
This is a great topic to discuss in our upcoming session, will proceed according to that.